lib/liquid/tags/raw.rb

```diff
@@ -12,29 +12,22 @@ def initialize(tag_name, markup, parse_context)
     end
 
     def parse(tokens)
-      @body = +''
-      while (token = tokens.shift)
-        if token =~ FullTokenPossiblyInvalid && block_delimiter == Regexp.last_match(2)
```

Comment:
Should remove `FullTokenPossiblyInvalid` from this class, since it's now in `BlockBody`